components/trust-manager/development/trust-manager-helm-generator.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: builtin
+kind: HelmChartInflationGenerator
+metadata:
+  name: trust-manager
+name: trust-manager
+repo: https://charts.jetstack.io
+version: 0.19.0
+namespace: cert-manager
+releaseName: trust-manager
+valuesInline:
+  resources:
+    requests:
+      cpu: 100m
+      memory: 128Mi
+    limits:
+      cpu: 200m
+      memory: 256Mi
```

Comment:
I think we're going with requests == limits everywhere for quality of service reasons.
I think we should try making those lower though, because we only have one certificate and namespaces are not being created all the time.
I'd say, let's start with `25m` and `64Mi` for requests and limits for the controller and the initContainer and monitor resource consumption.